src/general/smoothing_kernels.jl

```diff
@@ -400,7 +401,7 @@ end
 end
 
 @inline normalization_factor(::WendlandC2Kernel{2}, h) = 7 / (pi * h^2)
-@inline normalization_factor(::WendlandC2Kernel{3}, h) = 21 / (2pi * h^3)
+@inline normalization_factor(::WendlandC2Kernel{3}, h) = 21 / (pi * h^3 * 2)
```

Comment:
Can you please keep the multiplication in front of pi this is written like this everywhere. I am not sure why this was changed to remove the '*'...

Reply:
```julia
julia> typeof(2 * pi * 1f0)
Float64

julia> typeof(pi * 1f0 * 2)
Float32
```

Reply:
Than add a comment please... because this is stupid